### PR TITLE
Bug 1142732 - Support computed property names in object literals.

### PR DESCRIPTION
**Pull request by kmaglione**

### CODE REVIEW

File: validator/testcases/regex.py [hunk 1]
```diff
@@ -233,14 +233,12 @@ def applicable(cls, err, filename, document):
     def js_tests(self):
         title = "Marionette access is disallowed"
         for ref, bug in self.MARIONETTE_REFERENCES.items():
-            print ref
```
Comment (magopian) [inline validator/testcases/regex.py]: :+1:

**Comment by magopian** [comment]
Have you tried this on the code? Could you please add a test to the PR demonstrating this? Thanks!

**Comment by mstriemer** [comment]
Is there a test you can add for this? I'm hitting the error this fixes in https://github.com/mozilla/amo-validator/pull/274 but that seems like more of a side-effect than a real test.

**Comment by kmaglione** [comment]
I have tested it, but I'll add an actual test.

**Comment by kmaglione** [comment]
Added test.

File: jetpack/addon-sdk [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 74d7ae8f5496979baf1b7e995b38474dc81d5e70
+Subproject commit e2b154aff5c452d3b4f25bd67513122bd6172319
```
Comment (mstriemer) [inline jetpack/addon-sdk]: Does this need to change? Looks unrelated.

**Comment by mstriemer** [comment]
Thanks for adding the test. I'm r+ other than the addon-sdk change, is that needed?

**Comment by kmaglione** [comment]
Ugh. I didn't mean to add that. I made the changes on the wrong branch. I guess `stash` added to the index.

**Comment by kmaglione** [comment]
Fixed SDK submodule change.

**Comment by mstriemer** [comment]
Looks like the change to the `jsshell` version is missing, was that in here before? Something like https://github.com/mozilla/amo-validator/pull/274/files#diff-354f30a63fb0907d4ad57269548329e3R10.

**Comment by kmaglione** [comment]
Ugh. wth...

**Comment by kmaglione** [comment]
OK, I think it's all there now.
